### PR TITLE
Add API for list of agents

### PR DESCRIPTION
**Pull request by mgrachev**

### CODE REVIEW

**Review by khodzha — APPROVED** [review]

File: src/app/router.rs [hunk 1]
```diff
@@ -17,9 +19,14 @@ pub(crate) fn new(state: State, authn: svc_authn::jose::ConfigMap) -> Router {
 }
 
 fn api_router() -> Router {
-    Router::new() // TODO: ULMS-1743
+    Router::new()
+        .metered_route("/api/v1/healthz", get(v1::healthz))
+        .metered_route(
+            "/api/v1/classrooms/:classroom_id/agents",
+            get(v1::classroom::list_agents::<AppState>),
```
Comment (khodzha) [inline src/app/router.rs:26]: we need to add `.options` handler here to handle CORS so clients could do requests for themselves

**Review by khodzha — CHANGES_REQUESTED** [review]

File: src/app/router.rs (hunk 1 above)
Comment (mgrachev) [reply src/app/router.rs]: Done.

Comment (khodzha) [reply src/app/router.rs]: you'll need this layer, it adds cors headers
https://github.com/foxford/dispatcher/blob/3b2b36b6090d39af06235995c2264cb5cbbc4c2b/src/app/http.rs#L71

Comment (mgrachev) [reply src/app/router.rs]: Added.

File: src/db/agent_session.rs [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+    pub async fn execute(&self, conn: &mut PgConnection) -> sqlx::Result<Vec<Agent>> {
+        sqlx::query_as!(
+            Agent,
+            r#"
+            SELECT
+                agent_id AS "id: AgentId"
+            FROM agent_session
+            WHERE
+                classroom_id = $1::uuid
```
Comment (manifest) [inline src/db/agent_session.rs:29]: Do we need a `LIMIT` here?

**Review by manifest — APPROVED** [review]

**Review by manifest — APPROVED** [review]

File: src/db/agent_session.rs (hunk 2 above)
Comment (mgrachev) [reply src/db/agent_session.rs:29]: This needs to be discussed. If necessary, I'll add it in the next PR.

**Review by khodzha — APPROVED** [review]

**Pull request merged**